pkgs/development/python-modules/ttfautohint-py/default.nix

```
@@ -23,10 +24,20 @@ buildPythonPackage rec {
     hash = "sha256-NTog461RpyHKo/Qpicj3tflehaKj9LlZEN9qeCMM6JQ=";
   };
 
-  postPatch = ''
-    substituteInPlace src/python/ttfautohint/__init__.py \
-      --replace-fail 'find_library("ttfautohint")' '"${lib.getLib ttfautohint}/lib/libttfautohint${stdenv.hostPlatform.extensions.sharedLibrary}"'
-  '';
+  postPatch =
+    [
+      ''
+        substituteInPlace src/python/ttfautohint/__init__.py \
+          --replace-fail 'find_library("ttfautohint")' '"${lib.getLib ttfautohint}/lib/libttfautohint${stdenv.hostPlatform.extensions.sharedLibrary}"'
+
+      ''
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isLinux [
+      ''
+        substituteInPlace src/python/ttfautohint/memory.py \
+          --replace-fail 'find_library("c")' '"${glibc}/lib/libc.so.6"'
+      ''
+    ];
```

```suggestion
```

If there's a problem with the dependencies, they need to be fixed in those packages.

---

This is the correct place to fix it, but the way it was fixed would always use glibc and not the one of the current stdenv.

Should probably look something like this.

```suggestion
  postPatch = ''
    substituteInPlace src/python/ttfautohint/__init__.py \
      --replace-fail 'find_library("ttfautohint")' '"${lib.getLib ttfautohint}/lib/libttfautohint${stdenv.hostPlatform.extensions.sharedLibrary}"'
    substituteInPlace src/python/ttfautohint/memory.py \
      --replace-fail 'find_library("c")' '"${lib.getLib stdenv.cc.libc}/lib/libc${stdenv.hostPlatform.extensions.sharedLibrary}"'
  '';
```

---

Fixed in #378514